bundles/python/python_lexer.moon

```diff
@@ -46,12 +46,16 @@ howl.aux.lpeg_lexer ->
   operator = c 'operator', S'+-*/%~&^=!<>;:,.(){}[]|`'
 
   name = (alpha + '_')^1 * (alpha + digit + P'_')^0
+  type_name = c 'type', (upper^1 * (alpha + digit + '_')^0)
```

Comment:
OK that's a good point - should I reduce this to just apply to the 'class' statement, or just leave it out?